Sources/ArcGISToolkit/Documentation.docc/Tutorials/JobManagerTutorial.tutorial

```diff
@@ -0,0 +1,138 @@
+                
+                @Code(name: "JobManagerTutorialApp.swift", file: JobManagerAppStep1)
+            }
+            
+            @Step {
+                Adding background tasks begins with some project setting. Enable the kind of background processing needed by the app in the Background Modes. The job manager uses "Background fetch".
+                
+                The job manager registers a background task to allow all of it's jobs to check the status of the work happening on the servers. Getting the status is a lightweight operation that requires "Background fetch" mode.
+                
+                @Image(source: JobManagerTutorialBackgroundFetch, alt: "An image of the Background Modes settings with the Background fetch mode checked.")
+            }
+            
+            @Step {
+                Now tell the background task scheduler the identifier of the task that we want to run in the background. This is done in the project info for the app target. Add "com.esri.ArcGISToolkit.jobManager.statusCheck" as the first item in the Permitted Background Task Scheduler Identifiers.
+                
+                The job manager will register a background task with this identifier and submit a request to be allotted periodic background processing time. In this tutorial we will ask to be given time every 30 seconds. When the app gets background time is at the operating system's discretion; it may or may not be anywhere close to 30 seconds.
+                
+                @Image(source: JobManagerTutorialPermittedIdentifiers, alt: "An image of the list of Permitted Background Task Scheduler Identifiers.")
+            }
+            
+            @Step {
+                The code needed for setting up the background task tells the job manager how often it should ask to be woken up to check status.
+                
+                Add an init to JobManagerTutorialView. In the init, use preferredBackgroundStatusCheckSchedule on the job manager to set the job manager to request to be woken up every 30 seconds. This setting along with the two project settings complete the set up necessary for the job manager to check the status of jobs when the app is backgrounded.
+                
+                @Code(name: "JobManagerTutorialView.swift", file: JobManagerViewStep4, previousFile: JobManagerViewStep3)
+            }
+            
+            @Step {
+                Now we'll turn our attention to handling the app getting terminated while jobs are running. In the tutorial app, add code to a task that will run if the app gets launched because a download finished.
```

Comment:
```suggestion
                The operating system may determine to terminate your app if the app is suspended. Now we'll turn our attention to handle that situation while jobs are running. In the tutorial app, add code to a task that will run if the app gets launched because a download finished.
```